lib/govuk_app_config/railtie.rb

```diff
@@ -9,7 +9,7 @@ class Railtie < Rails::Railtie
     end
 
     config.before_initialize do
-      GovukLogging.configure if Rails.env.production?
+      GovukLogging.configure if ENV["ENABLE_PRODUCTION_LIKE_LOGS"]
```

Comment:
```suggestion
      GovukLogging.configure if ENV["GOVUK_RAILS_JSON_LOGGING"]
```

I think we should name this more based on what it does and we can have the naming [be consistent with govuk_sidekiq](https://github.com/alphagov/govuk_sidekiq/blob/26f3ff2cf500f507fc9b34089c811728365df627/lib/govuk_sidekiq/sidekiq_initializer.rb#L22) - which feels a bit more 12 factor.

I imagine a cherry on top would be to rename the class to GovukJsonLogging which would reflect the changes of https://github.com/alphagov/govuk_app_config/pull/300 where GovukLogging is now only setting up JSON logging